# You can find control table of Dynamixel on emanual (http://emanual.robotis.com/#control-table)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
# This example is compatible with Dynamixel using protocol 1.0
head_yaw_joint:
  ID: 30
  Return_Delay_Time: 0
  CW_Angle_Limit: 0
  CCW_Angle_Limit: 0
  Moving_Speed: 0
head_pitch_joint:
  ID: 31
  Return_Delay_Time: 0
  CW_Angle_Limit: 0
  CCW_Angle_Limit: 0
  Moving_Speed: 0